ROADMAP.md

```diff
@@ -1,25 +0,0 @@
-This repository is in active development as we strive to implement features outlined in the [GraphQL Spec](https://spec.graphql.org/).
-
-While this is a non-exhaustive list, current large features we are planning on building are as follows:
-
-1. **Semantic Analysis and Validation**
-    We will be building a separate structure on top of the the CST created by
-    the parser to allow for more efficient schema querying. This will include more
-    easily accessible information about queries, mutations, types and their
-    subtypes, directives etc.
-
-    Validation will correspond to [#Validation part](https://spec.graphql.org/October2021/#sec-Validation)
-    of the GraphQL spec. This feature allows us to have spec-correct queries and
-    schemas.
-
-3. **Execution.**
-    Corresponds to [#Execution part](https://spec.graphql.org/October2021/#sec-Execution)
-    of the GraphQL spec.  Execution has to be implemented after we implement
-    validation and HIR. It will run on Operations, SelectionSets, Fields, and
-    Requests.
```

Comment:
We *do* have most of a GraphQL execution engine in apollo-compiler, but only as part of schema introspection. It’s possible more of it will be made public in some form at some point, but at the moment we don’t really know what that would look like nor have any concrete plan to work in that direction.